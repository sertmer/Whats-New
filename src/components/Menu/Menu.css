.menu {
  width: 15%;
  height: 100%;
  display: flex;
  flex-direction: column;
}

.menu-btn {
  height: 2%;
  border-radius: 0 50px 50px 0;
  color: #ffffff;
  background-color: #141527;
  border-color: #141527;
  font-size: larger;
  margin: 5%;
  
}

.menu-btn:hover {
  border-color: #f08623;
  background-color: #f08623;
}