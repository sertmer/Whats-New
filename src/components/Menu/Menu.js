import React from 'react';
import './Menu.css'

const Menu = ({ changeCurrentPage, local, entertainment, health, science, technology }) => {
  return (
    <aside className='menu'>
      <h1>What's <span>New?</span></h1>
      <button 
        className='menu-btn' 
        onClick={() => changeCurrentPage(local)}>
        Local News
      </button>
      <button 
        className='menu-btn' 
        onClick={() => changeCurrentPage(technology)}>
        Technology
      </button>
      <button 
        className='menu-btn' 
        onClick={() => changeCurrentPage(entertainment)}>
        Entertainment
      </button>
      <button 
        className='menu-btn' 
        onClick={() => changeCurrentPage(science)}>
        Science
      </button>
      <button 
        className='menu-btn' 
        onClick={() => changeCurrentPage(health)}>
        Health
      </button>
    </aside>
  )
}

export default Menu;