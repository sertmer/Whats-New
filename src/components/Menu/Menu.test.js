import React from 'react'
import Menu from './Menu'
import { shallow } from 'enzyme'

describe('Menu', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Menu 
      changeCurrentPage={jest.fn()}
      />)

    expect(wrapper).toMatchSnapshot()
  })
})