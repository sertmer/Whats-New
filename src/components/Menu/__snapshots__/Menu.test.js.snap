// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match the snapshot 1`] = `
<aside
  className="menu"
>
  <button
    className="menu-btn"
    onClick={[Function]}
  >
    Local News
  </button>
  <button
    className="menu-btn"
    onClick={[Function]}
  >
    Technology
  </button>
  <button
    className="menu-btn"
    onClick={[Function]}
  >
    Entertainment
  </button>
  <button
    className="menu-btn"
    onClick={[Function]}
  >
    Science
  </button>
  <button
    className="menu-btn"
    onClick={[Function]}
  >
    Health
  </button>
</aside>
`;
